src/parser.rs

```diff
@@ -67,12 +68,8 @@ fn initialize_launch(
         launch.opt_rootfs_lxd.as_ref().map(|r| r.as_str()),
     )?;
 
-    Ok(Launch::new(
-        String::from(launch.container_id_or_name),
-        rootfs,
-        String::from(launch.name),
-        launch.cmd,
-    ))
+    let container = container::Container::new(&launch.container_id_or_name)?;
+    Launch::new(container, rootfs, String::from(launch.name), launch.cmd)
```

Comment:
`Ok`で包んでやるほうがわかりやすいと思った。

```suggestion
    Ok(Launch::new(container, rootfs, String::from(launch.name), launch.cmd)?)
```